assets/js/stripe-admin.js

```diff
@@ -59,6 +59,21 @@ jQuery( function( $ ) {
 					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).hide();
 				}
 			} ).change();
+
+			// Make the 3DS notice dismissable.
+			$( '.wc-stripe-3ds-missing' ).each( function() {
+				var $setting = $( this );
+
+				$setting.find( '.notice-dismiss' ).on( 'click.wc-stripe-dismiss-notice', function() {
+					$.ajax( {
+						type: 'post',
```

Comment:
Using a `HEAD` request with the nonce as a querystring parameter would be much more efficient.